doc/kubevela-definitions/cardano/node-connector.md

```diff
@@ -0,0 +1,109 @@
+# Cardano Node Connector trait
+
+## Overview
+
+The Cardano Node Connector in Kubvela acts as a bridge between your application and the Cardano node through specific application characteristics and settings
+
+The ***required*** parameter for implementing this trait is:
```

Comment:
How is it required if it has a default?

Reply:
I believe it's just a terminology confusion. The "default" value here is just for VelaUX , when someone is trying to configure the trait using the UI, "preview" is the first value available on the switchBox.